stdlib/public/core/Indices.swift

```diff
@@ -130,5 +130,12 @@ extension Collection where Indices == DefaultIndices<Self> {
   }
 }
 
-public typealias DefaultBidirectionalIndices<T: BidirectionalCollection> = DefaultIndices<T>
-public typealias DefaultRandomAccessIndices<T: RandomAccessCollection> = DefaultIndices<T>
+@available(*, deprecated, renamed: "DefaultIndices")
+public typealias DefaultBidirectionalIndices<
+  T : BidirectionalCollection
+> = DefaultIndices<T>
```

Comment:
I feel like you're just 80-col-trolling me now...

Reply:
Reverted. It doesn't make much sense here.